integration-tests/suites/common/collector_manager.go

```diff
@@ -36,15 +36,15 @@ func NewCollectorManager(e Executor, name string) *CollectorManager {
 		collectionMethod = "kernel_module"
 	}
 
+	logLevel := ReadEnvVarWithDefault("COLLECTOR_LOG_LEVEL", "debug")
 	offlineMode := ReadBoolEnvVar("COLLECTOR_OFFLINE_MODE")
 
 	env := map[string]string{
 		"GRPC_SERVER":             "localhost:9999",
-		"COLLECTOR_CONFIG":        `{"logLevel":"debug","turnOffScrape":true,"scrapeInterval":2}`,
+		"COLLECTOR_CONFIG":        fmt.Sprintf(`{"logLevel":"%s","turnOffScrape":true,"scrapeInterval":2}`, logLevel),
```

Comment:
Ah yes! Constructing a json string with a print statement! This is what peak performance looks like /s

When can we get rid of the json string being passed as an environment variable that later gets turned to an argument to the collector binary?

Reply:
I did consider making a struct and populating it with defaults and then json.Marshal into a string to at least make it a little less hard-coded, but with the upcoming changes to the integration tests, I didn't think it was yet worth it.